src/star/star_solo/test.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+
+set -e
+
+## VIASH START
+meta_executable="target/docker/star/star_solo/star_solo"
+meta_resources_dir="src/star/star_solo"
+## VIASH END
+
+#########################################################################################
+
+## TODO: add star solo tests
+
+
+# helper functions
+assert_file_exists() {
+  [ -f "$1" ] || (echo "File '$1' does not exist" && exit 1)
+}
+assert_file_doesnt_exist() {
+  [ ! -f "$1" ] || (echo "File '$1' exists but shouldn't" && exit 1)
+}
+assert_file_empty() {
+  [ ! -s "$1" ] || (echo "File '$1' is not empty but should be" && exit 1)
+}
+assert_file_not_empty() {
+  [ -s "$1" ] || (echo "File '$1' is empty but shouldn't be" && exit 1)
+}
+assert_file_contains() {
+  grep -q "$2" "$1" || (echo "File '$1' does not contain '$2'" && exit 1)
```

Comment:
Using `()` executes the command in a new shell; using `{;}` doesn't. Please note the `;` at the end of the grouped command

Reply:
wdym 'note the ; at the end of the grouped command'?

Reply:
I had changed it to something without () but then it didn't want to work anymore, so I dropped these changes :sweat_smile:

Reply:
This should work works:
```
[ -s "$1" ] || { echo "File '$1' is empty but shouldn't be" && exit 1; }
```
But this does not:

This should work works:
```
[ -s "$1" ] || { echo "File '$1' is empty but shouldn't be" && exit 1 }
```
(missing `;` at the end)